tests/jballerina-unit-test/src/test/resources/test-src/expressions/mappingconstructor/mapping_constructor_infer_record.bal

```diff
@@ -231,30 +231,26 @@ type Rec1 record {
     boolean b;
 };
 
-type Rec2 record {
-    string i?;
+type Rec2 record {|
+    string i;
     float f?;
-};
+|};
 
 function testInferredRecordTypeWithOptionalTypeFieldViaSpreadOp() {
-    Rec1 rec1 = {i: 1, b: true};
```

Comment:
Shall we remove the `Rec1` type definition if it is no longer used? And rename `Rec2` to `Rec1` maybe?